tests/integration.spec.ts

```diff
@@ -108,6 +108,28 @@ describe('jmap-client-ts', () => {
     expect(emailGetResponse.accountId).toBeDefined();
   });
 
+  it('should have email_changes working', async () => {
+    const getResponse = await client.email_get({
+      accountId: client.getAccountIds()[0],
+      ids: [],
+    });
+
+    const changesResponse = await client.email_changes({
+      accountId: client.getAccountIds()[0],
+      sinceState: getResponse.state,
+    });
+
+    expect(changesResponse).toMatchObject<IEmailChangesResponse>({
+      accountId: client.getAccountIds()[0],
+      oldState: getResponse.state,
+      newState: getResponse.state,
+      hasMoreChanges: false,
+      created: [],
+      updated: [],
+      destroyed: [],
+    });
```

Comment:
Can we have a test with actual changes?

Reply:
It was just a test to check the request works, with right attributes.

Reply:
Well I can't tell, as is that the fields where hardcoded to empty, for instance.